plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -156,7 +162,8 @@ func (adx *AzureDataExplorer) pushMetrics(ctx context.Context, format ingest.Fil
 	if err != nil {
 		return err
 	}
-
+	length := len(metricsArray)
+	adx.Log.Debugf("Metrics array length %d for table %s", length, tableName)
```

Comment:
This is very technical. How about
```suggestion
	adx.Log.Debugf("Writing %s metrics to table %q", length, tableName)
```